pkg/ratelimit/config.go

```diff
@@ -71,9 +71,8 @@ func RateLimiterFor(ctx context.Context, c *Config) (limiter.Store, error) {
 		}
 
 		return redisstore.NewWithPool(config, &redis.Pool{
-			DialContext: func(ctx context.Context) (redis.Conn, error) {
-				return redis.DialContext(ctx, "tcp", addr,
-					redis.DialUsername(c.RedisUsername),
```

Comment:
Hmm, I see open census doesn't have a dialusername, is that gonna be a problem? Should we patch opencensus to use that?

Reply:
Won't matter for us. Redis 6+ allows for user/pass, but we're on 5

Reply:
Yup, moreover the managed redis we provide on GCP actually don't support pass either.